src/Microsoft.AspNet.Mvc.Core/UnitTestability/MvcControllerUnitTestHelper.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Net.WebSockets;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Http.Security;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+
+// Not sure if this is the right location/namespace for this..
+namespace Microsoft.AspNet.Mvc.Core
```

Comment:
Discuss with @yishaigalatzer - my assumption is that we want another separate package - something like `Microsoft.AspNet.Mvc.Testing`. 

In general though there is no product code in the `Microsoft.AspNet.Mvc.Core` namespace. It should be `Microsoft.AspNet.Mvc` if it's something that a user will need to find easily, otherwise `Microsoft.AspNet.Mvc.<FeatureName>` if it's something obscure.